_kits/it/co-progettazione.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Co-progettazione
+subtitle: Coinvolgi stakeholder e utenti del servizio in una sessione di co-progettazione
+lang: it
+layout: kit
+description: "Un kit di Designers Italia dedicato alla progettazione partecipata di un servizio digitale"
+iconImage: "/assets/icons/kit/kit_coprog.svg"
+image: "/assets/icons/kit/social-card/kit_coprog.png"
+backgroundColor: "#439af1"
+nodate: true
+kitcategory: progettare
+order: 1
+redirect_from:
+  - /kit/co-design-workshop
+
+textChunk1:
+  title: A COSA SERVE
+  text: "<p>La co-progettazione è una metodologia fondamentale per la progettazione di servizi pubblici efficaci: ha l’obiettivo di raccogliere il punto di vista dei diversi attori coinvolti nell’erogazione e fruizione di un servizio, sollevando i problemi chiave e allo stesso tempo accelerando il processo di identificazione delle soluzioni.</p>
+  <p>Un workshop di co-progettazione è una <strong>sessione di lavoro di gruppo, durante la quale diversi soggetti (progettisti, utenti, stakeholder della Pubblica Amministrazione ed eventuali rappresentanti di aziende private) vengono invitati a discutere le criticità esistenti e individuare insieme delle possibili soluzioni progettuali, seguendo un percorso guidato da un facilitatore</strong>. L'attività di co-progettazione è preceduta da una serie di attività di ricerca e analisi del contesto, che aiutano a definire l’obiettivo del progetto su cui si sta lavorando e informano la sessione stessa.</p>
+  <p>Il kit contiene esempi di pianificazione e gestione di sessioni di co-progettazione focalizzate su tre tre casi d'uso: la progettazione di un servizio pubblico, di un <em>touchpoint</em> digitale (sito, app, piattaforma...),contenuti e linguaggio da usare nel servizio digitale. I materiali sono il punto di partenza che puoi elaborare e personalizzare a seconda dei tuoi obiettivi e dell’ambito specifico di intervento.</p>"
+
+guideLinesChunk:
+  title: GUIDA PRATICA
+  link: <a href='https://docs.italia.it/italia/designers-italia/design-linee-guida-docs/it/stabile/doc/service-design.html' target="_blank">Service design</a>
+
+rightColumnItems:
+  - title: KIT CORRELATI
+    items:
+      - <a href='https://designers.italia.it/kit/analisi-contesto/'>Analisi del contesto</a>
```

Comment:
Questi si potranno ottimizzare (con attenzione agli URL) con l'utilizzo della [funzione Slugify di Jekyll](https://jekyllrb.com/docs/liquid/filters/), in modo da inserire qui soltanto i titoli dei kit e tutto il resto potrebbe essere generato automaticamente.